### PR TITLE
Any Signer Stellar support 

### PR DESCRIPTION
**Pull request by EnoRage**
## Description

Add Stellar to Any Signer + Test to test/Any for it

## Testing instructions

Run tests

## Types of changes

Add code to src/Any/Signer.cpp
Add code to tests/Any/SignerTests.cpp
## Checklist

- [x] Prefix PR title with `[WIP]` if necessary.
- [x] Add tests to cover changes as needed.
- [x] Update documentation as needed.

### CODE REVIEW

File: src/Any/Signer.cpp [hunk 1]
```diff
@@ -210,4 +222,4 @@ void TW::Any::Signer::toJson(const google::protobuf::Message &message, std::stri
     options.preserve_proto_field_names = true;
 
     MessageToJsonString(message, json_string, options);
-}
+}
```
Comment (vikmeup) [inline src/Any/Signer.cpp]: can you add extra space here?

File: src/Any/Signer.cpp [hunk 2]
```diff
@@ -175,6 +176,17 @@ TW::Any::Proto::SigningOutput TW::Any::Signer::sign() const noexcept {
...
+        Stellar::Proto::SigningInput message;
+        parse(transaction, &message, output);
+        if (output.success()) {
+                message.set_private_key(privateKey.bytes.data(), privateKey.bytes.size());
+                auto signer = Stellar::Signer(message);
+                auto signerOutput = signer.sign();
+                output.set_output(signerOutput);
+        }
+        break;
+        }
```
Comment (vikmeup) [inline src/Any/Signer.cpp]: `}` to the left a bit

File: tests/Any/SignerTests.cpp [hunk 3]
```diff
@@ -233,3 +233,18 @@ TEST(Signer, NanoTransactionSign) {
...
+    input.set_private_key("59a313f46ef1c23a9e4f71cea10fc0c56a2a6bb8a4b9ea3d5348823e5a478722");
+    input.set_transaction(transaction);
+    input.set_coin_type(TWCoinTypeStellar);
+
+    auto signer = Signer(input);
+    auto output = signer.sign();
+
+    ASSERT_TRUE(output.success());
+    ASSERT_EQ(output.output(), "AAAAAAmpZryqzBA+OIlrquP4wvBsIf1H3U+GT/DTP5gZ31yiAAAD6AAAAAAAAAACAAAAAAAAAAIAAAAASZYC0gAAAAEAAAAAAAAAAQAAAADFgLYxeg6zm/f81Po8Gf2rS4m7q79hCV7kUFr27O16rgAAAAAAAAAAAJiWgAAAAAAAAAABGd9cogAAAEAOJ8wwCizQPf6JmkCsCNZolQeqet2qN7fgLUUQlwx3TNzM0+/GJ6Qc2faTybjKy111rE60IlnfaPeMl/nyxKIB");
+}
```
Comment (vikmeup) [inline tests/Any/SignerTests.cpp]: extra space needed

File: src/Any/Signer.cpp (hunk 1 above)
Comment (EnoRage) [reply src/Any/Signer.cpp]: Yep, added

File: src/Any/Signer.cpp (hunk 2 above)
Comment (EnoRage) [reply src/Any/Signer.cpp]: Done

File: tests/Any/SignerTests.cpp (hunk 3 above)
Comment (EnoRage) [reply tests/Any/SignerTests.cpp]: Done

**Pull request merged**